packages/plugin-amplitude-node/lib/index.ts

```diff
@@ -8,7 +8,7 @@ import Amplitude, { AmplitudeOptions } from 'amplitude';
 
 export { AmplitudeOptions };
 
-export default class AmplitudeNodePlugin extends PluginBase {
+export class AmplitudeNodePlugin extends PluginBase {
```

Comment:
To make importing this more uniform across platforms please remove the 'Node' and 'Browser' terms from the class names. To make it clear what the usage is please add a small comment block at the top of each class. E.g.
```
/**
 * Node Amplitude Plugin for Iteratively SDK
 */
export class AmplitudePlugin extends PluginBase {
 ...
} 
export default AmplitudePlugin;
```

Reply:
This should help clean up the codegen - https://bitbucket.org/seasyd/analytics_backend/pull-requests/285

Reply:
Fixed